Makefile

```diff
@@ -14,39 +15,30 @@ update_config=_config/update.yml
 scrape_config=_config/scrape.yml
 search_config=_config/search_index.yml
 
-# This target is invoked by a doc_independent job on the ROS buildfarm.
-html: build deploy
-
-build: download-previous discover update scrape search-index
+build: prepare-sources
+	bundle exec jekyll build --verbose --trace -d $(site_path) --config=$(config_file),$(index_file)
 
-download-previous:
+prepare-sources:
```

Comment:
Consider documenting what each target is supposed to do and how it should be used. This could be just comments in this file could be helpful, or an update to the README's "Skipping parts of the build".

Reply:
Uff that README is *really* outdated. PTAL